k8s/deploy.yaml

```
@@ -44,6 +44,9 @@ spec:
           env:
           - name: POLLING_ACTORS
             value: "1"
+          env:
```

YAMLlint (1.37.1)

[error] duplication of key "env" in mapping

(key-duplicates)